Assets/Scripts/Actors/ActorComponents/PlayerActorInventory.cs

```diff
@@ -25,6 +25,7 @@ public List<BuddyTag> buddies
 	GameObject heldResource;
 
 	ActorCamera _actorCamera;
+	PlayerActorStats _actorStats;
```

Comment:
Add this to player actor instead of in each of these scripts to reduce getcomponent calls and make things easier